File: app/src/main/java/org/stepik/android/exams/data/model/Dataset.java
package org.stepik.android.exams.data.model;


import java.util.List;

public final class Dataset {

    public Dataset(List<String> options, boolean is_multiple_choice) {
        this.options = options;
        this.is_multiple_choice = is_multiple_choice;
    }

    public Dataset(String string) {
        this.string = string;
    }

    private String string;
    private List<String> options;
    private boolean is_multiple_choice;

    public List<String> getOptions() {
        return options;
    }

    public boolean is_multiple_choice() {
        return is_multiple_choice;
    }
}

File: app/src/main/java/org/stepik/android/exams/data/model/Dataset.kt
package org.stepik.android.exams.data.model

import com.google.gson.annotations.SerializedName
import org.stepik.android.exams.data.model.attempts.Pair

data class Dataset(
        val options: List<String>? = null,
        val someStringValueFromServer: String? = null,
        val pairs: List<Pair>? = null,
        val rows: List<String>? = null,
        val columns: List<String>? = null,
        val description: String? = null,

        @SerializedName("is_multiple_choice")
        val isMultipleChoice: Boolean = false,
        @SerializedName("is_checkbox")
        val isCheckbox: Boolean = false,
        @SerializedName("is_html_enabled")
        val isHtmlEnabled: Boolean = false
)

class DatasetWrapper(val dataset: Dataset? = null)
